### PR TITLE
Fix code size bug

### PR DESCRIPTION
**Pull request by AlexExcellence**
- fixed a bug where the code size would jump between the default size (14) and the current binding size when using native selection for feedbacks 
- made syntax highlighting dynamic again

Before:

https://user-images.githubusercontent.com/79016456/215626614-71566dc5-7c2e-4bb9-8353-c670eedfa897.mov

After:

https://user-images.githubusercontent.com/79016456/215626679-b9e43d95-f359-4821-836e-f20e011fb2fe.mov

### CODE REVIEW

File: CodeEditor/Sources/CodeEditor/UXCodeTextView.swift
Comment (pal03377) [inline CodeEditor/Sources/CodeEditor/UXCodeTextView.swift]: I would suggest either removing the space padding or making it so that all of the dots align

```suggestion
            theme.codeFont = theme.codeFont?            .withSize(size)
            theme.boldCodeFont   = theme.boldCodeFont?  .withSize(size)
            theme.italicCodeFont = theme.italicCodeFont?.withSize(size)
```

File: CodeEditor/Sources/CodeEditor/UXCodeTextView.swift
Comment (pal03377) [inline CodeEditor/Sources/CodeEditor/UXCodeTextView.swift]: spelling: `drawHighlights` (one missing h)

File: CodeEditor/Sources/CodeEditor/UXCodeTextView.swift
Comment (pal03377) [inline CodeEditor/Sources/CodeEditor/UXCodeTextView.swift:444]: There will be merge conflicts with #96, which I think we should merge first. This comment is to prevent accidentally merging before we merged #96.

**Review by pal03377 — APPROVED** [review]
Looks fine & works nicely, good job! I tried merging this with #96 locally and there were some merge conflicts. Because I think we need #96 more urgently for the CAT Dry Run, we should merge it first and resolve any conflicts here.

File: CodeEditor/Sources/CodeEditor/UXCodeTextView.swift
Comment (pal03377) [reply CodeEditor/Sources/CodeEditor/UXCodeTextView.swift:444]: #96 is now merged

**Review by huber-florian — APPROVED** [review]
Please do a proper code formatting, but apart from that great work! Very nice!

**Comment by AlexExcellence** [comment]
Will merge when TestServer works again

**Pull request merged**
